tron/core/actionrun.py

```diff
@@ -733,27 +735,31 @@ def stop(self):
         return self._kill_mesos_task()
 
     def kill(self, final=True):
-        if not self.is_active:
-            return f'Action is {self.state}, not running.'
-
         if self.retries_remaining is not None and final:
             self.retries_remaining = -1
 
         if self.cancel_delay():
             return
 
         error_message = self._kill_mesos_task()
-        if error_message is not None:
+        if error_message:
             return error_message
         return "Warning: It might take up to docker_stop_timeout (current setting is 2 mins) for killing."
 
     def _kill_mesos_task(self):
+        msgs = []
+        if not self.is_active:
+            msgs.append(f'Action is {self.state}, not running. Sending kill anyway.')
+
         mesos_cluster = MesosClusterRepository.get_cluster()
         if self.mesos_task_id is None:
-            return "Error: Can't find task id for the action."
-        succeeded = mesos_cluster.kill(self.mesos_task_id)
-        if not succeeded:
-            return "Error while killing task. Please try again."
+            msgs.append("Error: Can't find task id for the action.")
+        else:
+            succeeded = mesos_cluster.kill(self.mesos_task_id)
+            if not succeeded:
+                msgs.append("Error while killing task. Please try again.")
```

Comment:
Can you also emit the positive feedback that it did work?
Normally I think it is OK for no-news to be good news, but with more rare-commands like kill, it is nice to know that it did work. Heck even emitting the actual taskid it killed would be nice

Reply:
Sure.